zokrates_cli/examples/book/assembly/division.zok

```diff
@@ -0,0 +1,7 @@
+def main(field a, field b) {
+    field mut c = 0;
+    asm {
+        c <-- a / b;
+        a === b * c;
```

Comment:
This is an example of the dangers of assembly: here you can set any value as the result of `0 / 0`
In the compiler, we first calculate `1 / b`, checked by `invb * b == 1` (which is unsatisfiable if `b == 0`)
Then return `invb * a`